figures/S9.supplemental_figure9/scripts/S9.r

```diff
@@ -203,6 +205,7 @@ fig5_probabilities <- (
     + ridge_plot_apoptosis
     + ridge_plot_pyroptosis
     + plot_layout(design = layout)
+    + plot_annotation(tag_levels = "A") &  theme(plot.tag = element_text(size = 20))
```

Comment:
This change was due to what looks like a single additional space. If possible in the future, consider the use of a style/formatter tool to help keep things consistent (and avoid the changed lines). Specifically, maybe [this pre-commit check could help via applying `styler`](https://lorenzwalthert.github.io/precommit/articles/available-hooks.html#style-files).

Reply:
Maybe the way lab needs a standard R style?